aws/studio_hpo_container_setup.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Really would want to remove extra text here and just focus on building the container</p><p> </p><p>Delete the image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>